ush/ioda/bufr2ioda/marine/b2i/bufr2ioda_insitu_profile_tropical.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+import sys
+from b2iconverter.util import parse_arguments
+from b2iconverter.bufr2ioda_config import Bufr2iodaConfig
+from b2iconverter.bufr2ioda_converter import Bufr2ioda_Converter
+from tropical_ioda_variables import TropicalIODAVariables
+
+
+platform_description = 'Tropical mooring profiles from dbuoy: temperature and salinity'
+
+
+class TropicalConfig(Bufr2iodaConfig):
+
+    def ioda_filename(self):
+        return f"{self.cycle_type}.t{self.hh}z.insitu_profile_tropical.{self.cycle_datetime}.nc4"
+
+
+if __name__ == '__main__':
+
+    script_name, config_file, log_file, test_file = parse_arguments()
+
+    bufr2ioda_config = TropicalConfig(
+        script_name,
+        config_file,
+        platform_description)
+
+    ioda_vars = TropicalIODAVariables()
+    ioda_vars.set_temperature_var_name("waterTemperature")
+    ioda_vars.set_temperature_error(0.02)
+    ioda_vars.set_salinity_var_name("salinity")
+    ioda_vars.set_salinity_error(0.01)
```

Comment:
Maybe not an issue for this PR, but I wonder if these errors should go in a config yaml?

Reply:
Sure, there are several such parameters that could be placed in the yaml.

Reply:
Also, to mention, these error coefficient values are much lower than what we are using for the other in situ observations.